src/UI/citizens/components/login/CitizensAdminLoginStyle.js

```diff
@@ -0,0 +1,121 @@
+import styled from 'styled-components';
+
+const LoginContainer = styled.div`
+  margin: auto;
+  width: 632px;
+  height: 264px;
+  border-radius: 4px;
+  border: solid 1px #d9d9d9;
+  background-color: #ffffff;
+`;
+
+const DataContainer = styled.div`
+  background-colour: yellow;
+  margin: 32px;
+`;
+
+const TitleAdminLogin = styled.div`
```

Comment:
zašto si postavljao tipgrafske property-je (font-family, font-size...) na div elementu?